includes/definitions/proxmox.yaml

```diff
@@ -0,0 +1,28 @@
+os: proxmox
+type: server
+group: unix
+text: proxmox pve
+icon: proxmox
+ifXmcbc: 1
+ifname: 1
+over:
+    - { graph: device_processor, text: 'Processor Usage' }
+    - { graph: device_ucd_memory, text: 'Memory Usage' }
+    - { graph: device_storage, text: 'Storage Usage' }
+poller_modules:
+    bgp-peers: 0
+    ospf: 0
+    stp: 0
+discovery_modules:
+    applications: 1
+    bgp-peers: 0
+    stp: 0
+    vmware-vminfo: 0
+    libvirt-vminfo: 0
+processor_stacked: 1
+discovery:
+    -
+      sysObjectId:
+            - .1.3.6.1.4.1.8072.3.2.10
+      sysDescr:
+            - PVE
```

Comment:
I think to be 100% safe here it might be better to do:

```
sysDescr_regex:
    - '/ PVE /'
```

Reply:
@laf just a note, this didn't need to be a regex `- ' PVE '` works too

Reply:
Good point